test/restart-failed.js

```diff
@@ -0,0 +1,54 @@
+'use strict';
+
+var debug = require('debug')('strong-cluster-control:test');
+var cluster = require('cluster');
+var control = require('../');
+
+var SIZE = global.SIZE;
+
+if (cluster.isWorker) {
+  if (cluster.worker.id > SIZE)
+    process.exit(3);
+  return;
+}
+
+var tap = require('tap');
+
+tap.test('good workers are not killed', function(t) {
+  var ended;
+
+  control.start({size: SIZE});
+
+  function size() {
+    return Object.keys(cluster.workers).length;
```

Comment:
I would probably just inline this in the 1 place it is called, but as `_.size(cluster.workers)`;